numba/parfors/array_analysis.py

```diff
@@ -1009,90 +1105,125 @@ def run(self, blocks=None, equiv_set=None):
         dprint_func_ir(self.func_ir, "before array analysis", blocks)
 
         if config.DEBUG_ARRAY_OPT >= 1:
-            print("ArrayAnalysis variable types: ", sorted(self.typemap.items()))
+            print(
+                "ArrayAnalysis variable types: ", sorted(self.typemap.items())
+            )
             print("ArrayAnalysis call types: ", self.calltypes)
 
         cfg = compute_cfg_from_blocks(blocks)
         topo_order = find_topo_order(blocks, cfg=cfg)
         # Traverse blocks in topological order
+        self._run_on_blocks(topo_order, blocks, cfg, init_equiv_set)
+
+        if config.DEBUG_ARRAY_OPT >= 1:
+            self.dump()
+            print(
+                "ArrayAnalysis post variable types: ",
```

Comment:
What's the "post" referring to and how is it different to https://github.com/numba/numba/pull/5320/files#diff-a46f9b8e7e173f928f6f58a2b9af202cR1109 ? (I know the answer, just wonder if we can make it clearer?).